app/src/main/sentences/sl/current_time.dslf

```diff
@@ -0,0 +1,2 @@
+current_time: high
+kaj|(koliko|je)|koliko je? ura (ali je)?;
```

Comment:
I am not sure what you mean here. `koliko` is present multiple times in the first OR group, and "je je ura ali je" would match, which seems strange. Maybe you meant `kaj|koliko je? ura (ali je)?` which would match all of: "kaj ura", "kaj ura ali je", "kaj je ura", "kaj je ura ali je", "koliko ura", "koliko ura ali je", "koliko je ura", "koliko je ura ali je". Is this better?